lib/trento/support/policy_helper.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Trento.Support.PolicyHelper do
+  @moduledoc """
+
+  Helper functions for bodyguard policies
+  """
+  alias Trento.Users.User
+
+  def user_has_ability?(%User{abilities: abilities}, %{name: name, resource: resource}),
```

Comment:
Could we have some basic UT specifically for this?

Reply:
done :P it was already tested but it's good to have dedicated tests

Reply:
thank you!